iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/SV.java

```diff
@@ -84,132 +74,341 @@ public double getA() {
         return a;
     }
 
-    private Complex computeU2(Complex y1, Complex y2, Complex z, double rho) {
-        Complex v1p = v1.multiply(rho); // v1p=v1*rho
-        Complex i1p = i1.divide(rho); // i1p=i1/rho
-
-        Complex i2p = i1p.subtract(y1.multiply(v1p)); // i2p=i1p-y1*v1p
-        Complex v2 = v1p.subtract(z.multiply(i2p)); // v2p=v1p-z*i2
-        Complex i2 = i2p.subtract(y2.multiply(v2)); // i2=i2p-y2*v2
-        Complex s2 = v2.multiply(3).multiply(i2.conjugate()); // s2=3*v2*conj(i2)
-        return v2.multiply(Math.sqrt(3f));
+    public int getEnd() {
+        return end;
     }
 
-    private Complex computeU2(Complex y, Complex z, double rho) {
-        return computeU2(y, new Complex(0, 0), z, rho);
+    /**
+     * @deprecated Not used anymore. Use
```

Comment:
Indicate that we deprecate this version with a _simplified_ view of the parameters of a generic branch to avoid misuse. I would enforce the explicit use of version that includes `rho` AND `alpha`.

Reply:
Done